ui-tests/tests/layer-browser.spec.ts

```diff
@@ -99,14 +99,14 @@ test.describe('#layerBrowser', () => {
   test.beforeAll(async ({ request }) => {
     const content = galata.newContentsHelper(request);
     await content.deleteDirectory('/testDir');
-    await content.uploadDirectory(
+    const uploaded = await content.uploadDirectory(
```

Comment:
Is this required?

Reply:
No, this is not. This is a left-over of some tests. I will remove it.